[2024-06-24T11:50:49.292+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T11:50:49.310+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T11:50:49.316+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T11:50:49.317+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-24T11:50:49.327+0700] {taskinstance.py:2330} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-06-23 00:00:00+00:00
[2024-06-24T11:50:49.332+0700] {standard_task_runner.py:63} INFO - Started process 30726 to run task
[2024-06-24T11:50:49.335+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'scheduled__2024-06-23T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp4jgx8e3p']
[2024-06-24T11:50:49.336+0700] {standard_task_runner.py:91} INFO - Job 10: Subtask is_api_available
[2024-06-24T11:50:49.392+0700] {task_command.py:426} INFO - Running <TaskInstance: user_processing.is_api_available scheduled__2024-06-23T00:00:00+00:00 [running]> on host daniel
[2024-06-24T11:50:49.477+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='is_api_available' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-23T00:00:00+00:00'
[2024-06-24T11:50:49.478+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T11:50:49.479+0700] {baseoperator.py:400} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2024-06-24T11:50:49.479+0700] {http.py:137} INFO - Poking: api/
[2024-06-24T11:50:49.491+0700] {base.py:84} INFO - Using connection ID 'user_api' for task execution.
[2024-06-24T11:50:50.046+0700] {base.py:294} INFO - Success criteria met. Exiting.
[2024-06-24T11:50:50.046+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T11:50:50.061+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, run_id=scheduled__2024-06-23T00:00:00+00:00, execution_date=20240623T000000, start_date=20240624T045049, end_date=20240624T045050
[2024-06-24T11:50:50.078+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T11:50:50.097+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24T11:50:50.099+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
